detection_rules/docs.py

```diff
@@ -24,6 +24,8 @@
 from .rule_loader import DeprecatedCollection, RuleCollection
 from .utils import load_etc_dump, save_etc_dump
 
+REPO_DOCS_DIR = "docs-dev"
```

Comment:
Do we still need this?

Reply:
I dont this constant is in need for this particular file. This is all external references to security-docs repo.